crates/protocol/genesis/Cargo.toml

```diff
@@ -22,6 +22,7 @@ alloy-consensus.workspace = true
 alloy-eips.workspace = true
 alloy-hardforks.workspace = true
 alloy-op-hardforks.workspace = true
+alloy-chains = { workspace = true, features = ["serde", "arbitrary"] }
```

Comment:
Let's break this out into the `arbitrary` and `serde` features. Shouldn't be enabling them by default.